sqldelight-gradle-plugin/src/test/kotlin/app/cash/sqldelight/tests/PluginTest.kt

```diff
@@ -40,6 +40,23 @@ class PluginTest {
     assertThat(result.output).contains("BUILD SUCCESSFUL")
   }
 
+  @Test
+  fun `Applying the plugin works fine for js projects`() {
+    val runner = GradleRunner.create()
+      .withCommonConfiguration(File("src/test/kotlin-js"))
+
+    val result = runner
+      .withArguments("clean", "generateMainDatabaseInterface", "--stacktrace")
+      .build()
+    assertThat(result.output).contains("BUILD SUCCESSFUL")
+
+    // Assert the plugin added the common dependency
+    val dependenciesResult = runner
+      .withArguments("dependencies", "--stacktrace")
+      .build()
+    assertThat(dependenciesResult.output).contains("app.cash.sqldelight:runtime")
```

Comment:
```suggestion
    assertThat(dependenciesResult.output).contains("app.cash.sqldelight:runtime:")
```
Otherwise we could have added `runtime-jvm` or something.